### PR TITLE
agenda/info updates

### PR DESCRIPTION
**Pull request by matt-long**

### CODE REVIEW

**Review by andersy005 — COMMENTED** [review]
👍 This is looking good! 

While you are at it, can you update the `views/Index.vue` with an entry for University of Washington?

```html
<h2 class="title">
              Dates: Oct. 16 - 18, 2019
              <br />Venues: <br />
              NCAR, Boulder, CO <br />
              LDEO, Palisades, NY
 </h2>
```

File: README.md [hunk 1]
```diff
@@ -3,3 +3,27 @@
 # CMIP6 Hackathon Main Website
 
 Website: [cmip6hack/cmip6hack.github.io](https://cmip6hack.github.io)
+
+
+# To build and test locally
+
+Create a conda environment
+```bash
+conda create -n web-dev -c conda-forge nodejs
```
Comment (andersy005) [inline README.md:12]: There are some pre-commit hooks for this repository. Can we update 

```bash
conda create -n web-dev -c conda-forge nodejs
conda activate web-dev
```
with 

```bash
conda create -n web-dev -c conda-forge nodejs pre-commit
conda activate web-dev
pre-commit install
```

Comment (matt-long) [reply README.md:12]: pre-commit install didn't work for me

Comment (andersy005) [reply README.md:12]: > pre-commit install didn't work for me

Interesting... What was the error?

**Review by andersy005 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (matt-long) [reply README.md:12]: pre-commit: command not found

**Pull request merged**